composer.json

```diff
@@ -137,9 +137,8 @@
   "require-dev": {
     "codeception/codeception": "^5.0.3",
     "codeception/module-symfony": "^3.1.0",
-    "codeception/phpunit-wrapper": "^9",
```

Comment:
`codeception/phpunit-wrapper` is included in `codeception/codeception` since version 5.0. This package is not installed because `codeception/codeception` have a `replaces` entry.